Bottom: 83e5afb8506ee89f3bc063c56fcb5a6f32a43862
Top:    35988cb00137c55b0074129532fb80e6fd9790d0
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-19 01:00:18 +0000

hid/gerber: Fix a couple of typos in drill_sort routine

Should fix sorting of drill holes by coordinate.

(Caught by clang)


---

diff --git a/src/hid/gerber/gerber.c b/src/hid/gerber/gerber.c
index 99f3185..5f5f349 100644
--- a/src/hid/gerber/gerber.c
+++ b/src/hid/gerber/gerber.c
@@ -628,8 +628,8 @@ drill_sort (const void *va, const void *vb)
   if (a->diam != b->diam)
     return a->diam - b->diam;
   if (a->x != b->x)
-    return a->x - a->x;
-  return b->y - b->y;
+    return a->x - b->x;
+  return a->y - b->y;
 }
 
 static int
